compass/landice/tests/ismip6_forcing/atmosphere/process_smb_racmo.py

```diff
@@ -0,0 +1,253 @@
+            os.makedirs(output_path)
+
+        src = os.path.join(os.getcwd(), output_file)
+        dst = os.path.join(output_path, output_file)
+        shutil.copy(src, dst)
+
+    def remap_source_smb_to_mali(self, input_file, output_file, mali_mesh_name,
+                                 mali_mesh_file, method_remap):
+        """
+        Remap the input ismip6 smb forcing data onto mali mesh
+
+        Parameters
+        ----------
+        input_file: str
+            input racmo smb data on its native rotaed pole grid
+        output_file : str
+            smb data remapped on mali mesh
+        mali_mesh_name : str
+            name of the mali mesh used to name mapping files
+        mali_mesh_file : str, optional
+            The MALI mesh file if mapping file does not exist
+        method_remap : str, optional
+            Remapping method used in building a mapping file
+
+        """
+        mapping_file = f"map_racmo_24km_to_" \
+                       f"{mali_mesh_name}_{method_remap}.nc"
+
+        # check if a mapfile exists
+        if not os.path.exists(mapping_file):
+            # build a mapping file if it doesn't already exist
+            self.logger.info(f"Creating a mapping file. "
+                             f"Mapping method used: {method_remap}")
+            build_mapping_file(self.config, self.cores, self.logger,
+                               input_file, mapping_file, mali_mesh_file,
+                               method_remap)
+        else:
+            self.logger.info("Mapping file exists. "
+                             "Remapping the input data...")
+
+        args = ["ncremap",
+                "-i", input_file,
+                "-o", output_file,
+                "-m", mapping_file,
+                "-v", "smb"]
+
+        subprocess.check_call(args)
+
+    def rename_source_smb_to_mali_vars(self, remapped_file_temp, output_file):
+        """
+        Rename variables in the remapped source input data
+        to the ones that MALI uses.
+
+        Parameters
+        ----------
+        remapped_file_temp : str
+            temporary ismip6 data remapped on mali mesh
+        output_file : str
+            remapped ismip6 data renamed on mali mesh
+        """
+
+        # open dataset in 20 years chunk
+        ds = xr.open_dataset(remapped_file_temp, chunks=dict(time=20),
+                             engine="netcdf4")
+
+        # build dictionary for ismip6 variables that MALI takes in
+        ismip6_to_mali_dims = dict(
+            time="Time",
+            ncol="nCells")
+        ds = ds.rename(ismip6_to_mali_dims)
+
+        # drop unnecessary variables
+        ds = ds.drop_vars(["height"])
+
+        # squeeze unnecessary coordinate variable
+        ds["sfcMassBal"] = ds["sfcMassBal"].squeeze(dim="height")
+
+        # write to a new netCDF file
+        write_netcdf(ds, output_file)
+        ds.close()
+
+    def correct_SMB_anomaly_for_baseSMB(self, output_file, mali_mesh_file):
```

Comment:
```suggestion
    def correct_smb_anomaly_for_base_smb(self, output_file, mali_mesh_file):
```
Method and function names should be all lowercase.